python/neuroglancer/lib.py

```diff
@@ -46,6 +55,25 @@ def helper(elem, shape, i):
 
     return helper(elem, shape, 0)
 
+
+def find_closest_divisor(to_divide, closest_to):
```

Comment:
no problem. i just need some documentation so i know how to run this function so i can test it out.